components/testimonial/Testimonial.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './Testimonial.module.scss';
+
+function Testimonial() {
+	return (
+		<div className={styles.container}>
+			<div className={styles.header}>
+				<img
+					className={styles.quotation_mark}
+					src="/images/quotation-mark.svg"
+					alt="Large quotation mark"
+				/>
+				<h4>Testimonial</h4>
+			</div>
+			<p>
+				Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolor, neque
+				aspernatur. Quis voluptas numquam sequi iste qui hic rerum, quod optio
+				eum at repellendus esse, similique, illum fuga tempore repudiandae.
+			</p>
+
+			<div className={styles.author}>
+				<img
+					className={styles.authorImage}
```

Comment:
let's not add this image to our images folder since it's not a true asset that we will be keeping.  Lets instead use an existing image in our assets so we're not adding too much to our bundle size unnecessarily